scripts/generate-llms-txt.js

```diff
@@ -0,0 +1,265 @@
+  }
+}
+
+// Function to map special cases for file names that don't match the sidebar
+function mapDocPath(item) {
+  const specialCases = {
+    'environment-setup': 'getting-started.md',
+    'native-platform': 'native-platforms.md',
+    'turbo-native-modules-introduction': 'turbo-native-modules.md',
+    'fabric-native-components-introduction': 'fabric-native-components.md',
+  };
+
+  if (typeof item === 'string') {
+    return specialCases[item] || `${item}.md`;
+  } else if (item.type === 'doc' && item.id) {
+    return specialCases[item.id] || `${item.id}.md`;
+  }
+  return `${item}.md`;
+}
+
+// Function to generate markdown documentation
+function generateMarkdown(sidebarConfig) {
+  let markdown = `# ${TITLE}\n\n`;
+  markdown += `> ${DESCRIPTION}\n\n`;
+  markdown += `This documentation covers all aspects of using React Native, from installation to advanced usage.\n\n`;
+
+  // Process each section (docs, api, components)
+  Object.entries(sidebarConfig).forEach(([section, categories]) => {
+    markdown += `## ${section.charAt(0).toUpperCase() + section.slice(1)}\n\n`;
+
+    // Process each category within the section
+    Object.entries(categories).forEach(([categoryName, items]) => {
+      markdown += `### ${categoryName}\n\n`;
+
+      // Process each item in the category
+      items.forEach(item => {
+        if (typeof item === 'string') {
+          // This is a direct page reference
+          const docPath = `${DOCS_PATH}${mapDocPath(item)}`;
+          const title = extractTitleFromMarkdown(docPath);
+          markdown += `- [${title}](${URL_PREFIX}${item})\n`;
+        } else if (typeof item === 'object') {
+          if (item.type === 'doc' && item.id) {
+            // This is a doc reference with an explicit ID
+            const docPath = `${DOCS_PATH}${mapDocPath(item)}`;
+            const title = extractTitleFromMarkdown(docPath);
+            markdown += `- [${title}](${URL_PREFIX}${item.id})\n`;
+          } else if (item.type === 'category' && Array.isArray(item.items)) {
+            // This is a category with nested items
+            markdown += `#### ${item.label}\n\n`;
+            item.items.forEach(nestedItem => {
+              if (typeof nestedItem === 'string') {
+                const docPath = `${DOCS_PATH}${mapDocPath(nestedItem)}`;
+                const title = extractTitleFromMarkdown(docPath);
+                markdown += `- [${title}](${URL_PREFIX}${nestedItem})\n`;
+              } else if (nestedItem.type === 'doc' && nestedItem.id) {
+                const docPath = `${DOCS_PATH}${mapDocPath(nestedItem)}`;
+                const title = extractTitleFromMarkdown(docPath);
+                markdown += `- [${title}](${URL_PREFIX}${nestedItem.id})\n`;
+              }
+            });
+          }
+        }
+      });
+    });
+  });
+
+  // Add newlines after all markdown headers
+  markdown = markdown.replace(/(#+ .*)\n/g, '\n$1\n');
+
+  return markdown;
```

Comment:
```suggestion
  // Format and cleanup whitespaces
  return markdown.replace(/(#+ .*)\n/g, '\n$1\n').replace(/\n(\n)+/g, '\n\n');
```

This code was adding new lines before the header, not after. Also I have updated it remove duplicated whitespace lines.